nutils/evaluable.py

```diff
@@ -3046,6 +3055,8 @@ def __init__(self, angle:asarray):
     super().__init__(args=[angle], shape=(*angle.shape, 2), dtype=float)
 
   def _derivative(self, var, seen):
+    if self.dtype == complex:
+      raise NotImplementedError('The complex derivative is not implemented.')
```

Comment:
I know that supporting this has no conceivable practical relevance, but in this case supporting complex derivatives actually seems easier than to not do so. All it takes is to set self.dtype = angle.dtype.

Reply:
I've removed `TrigNormal` and `TrigTangent`.